style.css

```diff
@@ -12,6 +12,12 @@ body {
   justify-content: center;
 }
 
+.hotkeys {
```

Comment:
We might want to make it only show on hover, because on smaller screens it overlays the gradient mesh

Reply:
Or just toggleable with a button or something

Reply:
Good idea